todo_app/app.py

```diff
@@ -1,5 +1,7 @@
-from flask import Flask
-
+from flask import Flask, redirect, request
+from flask import render_template
```

Comment:
Only a matter of preference, but note that you have the option to combine these like the first line:
```suggestion
from flask import Flask, redirect, request, render_template
```